### PR TITLE
Added support to load MTs and other plastids with more features

### PR DESCRIPTION
**Pull request by ens-carlos**
Added support for different translation tables: removed codon_table parameter. This is now fetched from the relevant CDS feature "transl_table" attribute.
Added support for multi-exon genes (including reverse strand ones).
Skip 'trans_splicing' features (subject to https://github.com/Ensembl/ensembl-io/pull/117 approval).

Chloroplasts like https://www.ncbi.nlm.nih.gov/nuccore/NC_011032.1 can be loaded now.

This script could be renamed to something more generic to reflect it can load more types of plastids.

### CODE REVIEW

**Comment by thibauthourlier** [comment]
I don't think the codon table attribute should be move but because I won't have the time to follow this I'm not formally requesting a change. I also know that the API might go against my wishes...
I think the whole script needs to be rewritten as a module instead of hack fixing but also to remove some of the options we don't need anymore which might complicated the whole thing.

We probably should think of annotating MTs and other non nuclear sequences at one point. I don't remember what the browser agreement says about them.

**Comment by ens-carlos** [comment]
Just an update, the related pull-request [Ensembl/ensembl-io#117](https://github.com/Ensembl/ensembl-io/pull/117)

**Comment by thibauthourlier** [comment]
Has the script been tested with a MT?

**Comment by ens-carlos** [comment]
Yes, I tested it on the Texel sheep back in December.

**Review by thibauthourlier — APPROVED** [review]
As previously stated in the comments, I think more work is needed but that's probably for the python version

**Comment by thibauthourlier** [comment]
I'll wait for the ensembl-io PR to be merged before merging this one

**Pull request merged**
